listenstore/listenstore/listenstore.py

```diff
@@ -106,19 +110,18 @@ def execute(self, query, params={}):
             return res.fetchall()
 
     def fetch_listens_from_storage(self, user_id, from_id, to_id, limit, order, precision):
-        query = """ SELECT listen.id, user_id, extract(epoch from ts), artist_msid, album_msid, recording_msid, data
-                    FROM listen, listen_json
-                    WHERE listen.id = listen_json.id AND user_id = %(user_id)s AND extract(epoch from ts) > %(from_id)s
-                        AND extract(epoch from ts) < %(to_id)s
-                    ORDER BY extract(epoch from ts) """ + ORDER_TEXT[order] + """ LIMIT %(limit)s
-                """
-        params = {
+        results = self.execute("""
+            SELECT listen.id, user_id, extract(epoch from ts), artist_msid, album_msid, recording_msid, data
+              FROM listen, listen_json
+             WHERE listen.id = listen_json.id AND user_id = %(user_id)s AND extract(epoch from ts) > %(from_id)s
+               AND extract(epoch from ts) < %(to_id)s
```

Comment:
If you have to put one of the statements on a new line, make every statement in `WHERE` be on a new line too.